File: Some_cpp_sols/A270.cpp
#include<iostream>
#include<vector>
using namespace std;
int main(){
    int ans,angle,q;
    cin>>q;
    while(q--){cin>>angle;ans = 180 - angle;cout<<(360 % ans == 0 ? "YES\n": "NO\n");}
    return 0;
}

File: A/A270.java
import java.util.Scanner;

public class A270{
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int t = input.nextInt();
        int ans,angle;
        while(t--!=0){
            angle = input.nextInt();
            ans = 180 - angle;
            if(360 % ans == 0)
                System.out.println("YES");
            else
                System.out.println("NO");
        }
        input.close();
    }
}